packages/web-components/src/compound-button/compound-button.styles.ts

```diff
@@ -0,0 +1,121 @@
+import { css } from '@microsoft/fast-element';
+import { styles as ButtonStyles } from '../button/button.styles.js';
+import {
+  colorNeutralForeground2,
+  colorNeutralForeground2BrandHover,
+  colorNeutralForeground2BrandPressed,
+  colorNeutralForeground2Hover,
+  colorNeutralForeground2Pressed,
+  colorNeutralForegroundDisabled,
+  colorNeutralForegroundOnBrand,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontWeightRegular,
+  lineHeightBase300,
+  lineHeightBase400,
+  spacingHorizontalS,
+  spacingHorizontalSNudge,
+  spacingHorizontalXS,
+} from '../theme/design-tokens.js';
+
+// Need to support icon hover styles
+export const styles = css`
+  ${ButtonStyles}
+
+  :host .control,
+  :host(:is([size])) .control {
+    gap: 12px;
+    height: auto;
+    padding-top: 14px;
+    padding-inline: 12px;
+    padding-bottom: 16px;
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+  }
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    text-align: start;
+  }
+
+  ::slotted([slot='description']) {
+    color: ${colorNeutralForeground2};
+    line-height: 100%;
+    font-size: ${fontSizeBase200};
+    font-weight: ${fontWeightRegular};
+  }
+
+  :host ::slotted(svg),
+  :host([size='large']) ::slotted(svg) {
```

Comment:
Maybe it is just me, but the two selectors and `!important`s are difficult to understand. It would be great if we could be adding comments for complex selectors.

Reply:
Important wasn't necessary here